babelOptions.js

```diff
@@ -0,0 +1,9 @@
+exports.getBabelOptions = ({useESModules}) => ({
+  presets: [['@babel/env', {loose: true}], '@babel/flow', '@babel/react'],
+  plugins: [
+    ['@babel/proposal-class-properties', {loose: true}],
+    ['@babel/transform-runtime', {useESModules}],
+    ['@babel/plugin-proposal-export-namespace-from'],
```

Comment:
Is this plugin actually used?

Reply:
I searched for `export * as` - nothing found

Reply:
I guess we can remove it